schemas/content-fragments/hotels/overview/hotelOverview.ts

```diff
@@ -0,0 +1,28 @@
+export default {
+  Title: "Hotel Overview",
+  name: "hotelOverview",
+  type: "document",
+  fields: [
+    {
+      name: "title",
+      title: "Title",
+      type: "string",
+    },
+    {
+      name: "description",
+      title: "Description",
+      type: "string",
+    },
+    {
+      name: "bannerImage",
+      title: "Banner Image",
+      type: 'array',
+      of: [{ type: 'media' }]
+    },
+    {
+      name: "basicInfo",
```

Comment:
Same nomenclature for both type and name.

Reply:
resolved , please check

Reply:
Not resolved, Please recheck